ribasim_qgis/pyproject.toml

```
@@ -0,0 +1,32 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ribasim_qgis"
+description = "Ribasim QGIS Plugin"
+readme = "README.md"
+authors = [{ name = "Deltares and contributors", email = "ribasim.info@deltares.nl" }]
+license = { text = "MIT" }
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Hydrology",
+]
+requires-python = ">=3.10"
+dynamic = ["version"]
+
+[tool.setuptools]
+zip-safe = true
+
+[tool.setuptools.dynamic]
+version.attr = "ribasim_testmodels.__version__"
+
+[tool.setuptools.packages.find]
+include = ["*"]
+
+[tool.setuptools.package-data]
+"ribasim_testmodels" = ["py.typed"]
```

```suggestion
"ribasim_qgis" = ["py.typed"]
```

I think you also need to add a py.typed file like the other Python packages.